nixos/modules/services/networking/whoogle-search.nix

```diff
@@ -0,0 +1,77 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.whoogle-search;
+in
+{
+  options = {
+    services.whoogle-search = {
+      enable = lib.mkEnableOption "Whoogle, a metasearch engine";
+
+      port = lib.mkOption {
+        type = lib.types.port;
+        default = 5000;
+        description = "Port to listen on.";
+      };
+
+      listenAddress = lib.mkOption {
+        type = lib.types.str;
+        default = "127.0.0.1";
+        description = "Address to listen on for the web interface.";
+      };
+
+      httpsOnly = lib.mkOption {
+        type = lib.types.bool;
+        default = false;
+        description = "Enforce HTTPS redirects for all requests.";
+      };
```

Comment:
This option is a bit mysterious to me. I assume it is for reverse-proxy usage, right? Because I don't think Whoogle will serve both HTTP and HTTPS on the same port. But if it is for reverse-proxy usage, you can just configure a HTTPS redirect there...

I suppose we can keep it anyway.

Reply:
yeah, I think it's for reverse proxy usage: https://github.com/benbusby/whoogle-search#https-enforcement
Since there is no real use for it in NixOS, I think we should remove this option to avoid confusion.